src/components/menus/wallet_settings.vue

```diff
@@ -647,6 +727,32 @@ export default {
       this.modals.change_password.new_password = "";
       this.modals.change_password.new_password_confirm = "";
     },
+    async doExportTransfers() {
+      this.hideModal("export_transfers");
+
+      let passwordDialog = await this.showPasswordConfirmation({
+        title: this.$t("dialog.exportTransfers.title"),
+        noPasswordMessage: this.$t("dialog.exportTransfers.message"),
+        ok: {
+          label: "export".toLocaleUpperCase(this.locale),
+          color: "primary"
+        },
+        dark: this.theme == "dark",
+        color: this.theme == "dark" ? "white" : "dark"
+      });
+      passwordDialog
+        .onOk(password => {
+          // if no password set
+          password = password || "";
+          if (this.modals.key_image.type == "Export")
```

Comment:
Is this not `modals.export_transfers` ?

Reply:
Export transfers isn't working for me on Mac. When I click "Confirm" the popup just disappears as if it was dismissed and nothing happens. I'm gueessing the problem is in this area of the code somewhere.

Reply:
> Is this not `modals.export_transfers` ?
In fact, why is this `if` statement necessary at all? There is no toggle when exporting the transfers

Reply:
Yeah this if statement shouldnt be there. From the export key images modal which has an "Export" and "Import" radio button.